app/lib/appointment_importer.rb

```diff
@@ -0,0 +1,41 @@
+class AppointmentImporter
+  FAKE_DATE_OF_BIRTH = '1950-01-01'.freeze
```

Comment:
One thing we _could_ do here, is infer a different age if `Are you aged between 50-54` is true. This could be handy as they're a different type of appointment.